packages/governance-modules/cannon.json

```diff
@@ -0,0 +1,11 @@
+{
+  "chains": [
+    {
+      "deploy": [
+        "synthetix-governance:0.0.1"
```

Comment:
should this version be bounded to the `package.json` value?

Reply:
Yess, updating.

Reply:
Responded on next comment 👇